Apps/Apps.xcworkspace/contents.xcworkspacedata

```diff
@@ -13,4 +13,22 @@
    <FileRef
       location = "group:..">
    </FileRef>
+   <FileRef
+      location = "group:../TestPlans/MapboxMapsAll.xctestplan">
+   </FileRef>
+   <FileRef
+      location = "group:../TestPlans/Examples.xctestplan">
+   </FileRef>
+   <FileRef
+      location = "group:../TestPlans/StressTest.xctestplan">
+   </FileRef>
+   <FileRef
+      location = "group:../TestPlans/DebugApp.xctestplan">
+   </FileRef>
+   <FileRef
+      location = "group:../TestPlans/UnitTests.xctestplan">
+   </FileRef>
+   <FileRef
+      location = "group:../TestPlans/IntegrationTests.xctestplan">
+   </FileRef>
```

Comment:
We can avoid explicit references as the will be available through folder in SPM tree